src/System.Device.Gpio.Tests/GpioControllerTestBase.cs

```diff
@@ -20,6 +20,7 @@ public void ControllerCanTurnOnLEDs()
         {
             using (GpioController controller = new GpioController(GetTestNumberingScheme(), GetTestDriver()))
             {
+                Console.WriteLine("ControllerCanTurnOnLEDs run");
```

Comment:
looks like you got some remainder from debugging?

Reply:
Yeah 😛 I noticed that too, but didn't fixed it because still trying to debug/run testing, will be removed